styles.css

```diff
@@ -0,0 +1,104 @@
+:root {
+  --gold: #f0b106;
+}
+body {
+  margin: 0;
+  background-color: var(--bg-color);
+}
+.container {
+  background-color: var(--white);
+  color: var(--black);
+  max-width: 42rem;
+  margin: auto;
+  height: unset;
+  min-height: 100vh;
+  flex-direction: column;
+}
+.wrapper_main {
+  padding: 1rem 2rem;
+}
+header > .card {
+  background-color: var(--primary);
+  border-color: var(--primary-li);
+  color: var(--primary-li);
+  width: 100%;
+  margin: 1.5rem 0;
+}
+
+/* nav  */
+.container > .wrapper_nav {
+  justify-content: space-between;
+  height: 4rem;
+  padding: 0 2rem;
+}
+.nav_brand_text {
+  text-transform: uppercase;
+  cursor: pointer;
+  color: var(--bg-color);
+  font-family: "Shadows Into Light", cursive;
+  font-weight: 600 !important;
```

Comment:
avoid using !important instead you can play with specificity. but if you want to keep it as it is you can

Reply:
Updated! Thanks!!